src/views/evaluation/Evaluation.js

```diff
@@ -1,15 +1,15 @@
-const NAV = function (option = {}){
+const NAV = function (option = {}, logo = {}){
```

Comment:
Yes. Well my logic there is the next: I want option(s) to be a  list of object of exactly same type, and logo button in the <li> is somewhat very different, as it doesn't behave like the others and I made it mostly as a dummy. We were not instructed to make the button to act in certain way. SO in order to keep everything "organized" I decided to make a separate class for it, even with the though that if I create a new object at the end or something it will be ignore because it doesn't have the fields required like the other objects.